lib/amazon-pricing/ec2-dhi-price-list.rb

```diff
@@ -40,6 +40,7 @@ def os_types
       'm5d' => { "large"=>48, "xlarge"=>24, "2xlarge"=>12, "4xlarge"=>6, "12xlarge"=>2, "24xlarge"=>1 },
       'i2' => { "xlarge"=>8, "2xlarge"=>4, "4xlarge"=>2, "8xlarge"=>1, "16xlarge"=>1 },
       'i3' => { "large"=>32, "xlarge"=>16, "2xlarge"=>8, "4xlarge"=>4, "8xlarge"=>2, "16xlarge"=>1, "metal"=>1 },
+      't3' => { "nano"=> 64, "micro"=> 32, "small"=> 16, "medium"=> 8, "large"=> 4, "xlarge" => 2, "2xlarge"=> 1},
```

Comment:
unclear if this change is needed, i.e. is DH supported for t3 (as it was _not_ for t1/t2).

and t1/t2/t3 does _not_ show up here:  https://aws.amazon.com/ec2/dedicated-hosts/pricing/

Reply:
I would say we wait a day or so, since this instance JUST dropped, and see if they update. They might not have rolled out all changes to the website yet.